docs/database/hbase.md

```diff
@@ -40,10 +40,8 @@ This attribute has stability level RELEASE CANDIDATE.
 **[3]:** It is RECOMMENDED to capture the value as provided by the application
 without attempting to do any case normalization.
 
-A single database query may involve multiple operations. If the operation
-name is parsed from the query text, it SHOULD only be captured for queries that
-contain a single operation or when the operation name describing the
-whole query is available by other means.
+The operation name SHOULD NOT be extracted from `db.query.text`,
```

Comment:
I don't think it's applicable to HBASE (but as in other cases, this is not a new problem introduced in this PR)

Reply:
Tracking in #1573